doc/exts/pylint_messages.py

```diff
@@ -7,7 +7,7 @@
 import os
```

Comment:
This all seems to work.

The only thing I would add is to add all checkers that can emit a message to the page. So for these messages both `stdlib` and `imports`. But that can also be done in a follow-up.